src/Config.js

```diff
@@ -149,6 +149,10 @@ Config.prototype.initOptions = function () {
     this.opts.option('metatile', {
         help: 'Override mml metatile setting [Default: mml setting]'
     });
+    this.opts.option('style_name', {
```

Comment:
If we want to override `Project.id`, I'd not call the option `style_name`, but rather `style_id`, or maybe just `id`.

Reply:
Because from user's view id means nothing, but style name self described well.

Reply:
I'm afraid `name` is confusing with the `name` key of the mml/yml file.

Reply:
Maybe then style_id?

Reply:
Yes, that was one of my two suggestions ;)